### DISCUSSION TITLE
Can not delete User, dependent object found

### DISCUSSION THREAD

**Discussion by julianstolp** [netbox-community/netbox#15012] [Help Wanted!]
I wanted to delete some users in our testing instance but one throws an error `Unable to delete <prefix>. 1 dependent objects were found: <prefix>`. The user never had any write permissions. Where do I search for this constraint? And how could this happen?

**Comment by jeremystretch** [comment]
Is that the literal error message, or have you replaced some other value with `<prefix>`? There's no native association from a prefix to a user in NetBox that I can think of.

**Reply by julianstolp** [reply]
I replaced the actual prefix with `<prefix>`. Can I extract the values/associations from the database somehow? I'm curious to find the cause.
We use custom scripts and the api but none of it is affecting the user model in any way.

**Reply by julianstolp** [reply]
@jeremystretch FYI I upgraded to v3.7.2, and I could delete the users..

**Reply by thojohnston** [reply]
For what its worth, I also had the same issue on v3.7.1 and upgrading to v3.7.2 fixed it.

**Reply by candlerb** [reply]
Perhaps it's this from v3.7.2 release notes:

* [#14851](https://github.com/netbox-community/netbox/issues/14851) - Automatically remove any associated bookmarks when deleting a user
